notebooks/Demo.ipynb

```diff
@@ -22,7 +22,7 @@
     "import plotly.express as px\n",
     "import matplotlib.pyplot as plt\n",
     "from mpl_toolkits.axes_grid1 import make_axes_locatable\n",
-    "from tqdm.auto import tqdm"
+    "from tqdm import tqdm, tqdm_notebook"
```

Comment:
Found the [issue](https://github.com/tqdm/tqdm/issues/824) mentioning this warning. Despite this, I will stay with tqdm_notebook import since this file is a jupyter notebook :laughing: